contracts/eth/nearbridge/test/NearBridge2.js

```diff
@@ -43,23 +43,20 @@ it('should be ok', async function () {
 
 });
 
-it('2020-09-09 Example', async function () {
-   // Skip until tests are upgraded having blocks after nearcore 1.23.0
-   this.skip();
-
-   const block_15178713 = borshify(require('./block_15178713.json'));
-   const block_15178760 = borshify(require('./block_15178760.json'));
-   const block_15204402 = borshify(require('./block_15204402.json'));
-   const block_15248583 = borshify(require('./block_15248583.json'));
```

Comment:
We don't need this files anymore and should delete files: "block_15178713.json", "block_15178760.json", "block_15204402.json", "block_15248583.json"

Reply:
1c06fa0cd78cbacb414968d8857501058dc25e8a